tox.ini

```diff
@@ -74,7 +74,7 @@ requires = virtualenv<=20.2.1
 
 [testenv]
 # Wheels for gevent segfault pretty easily
-install_command=python -m pip install --no-binary gevent {opts} {packages}
+install_command=python -m pip install Cython {opts} {packages}
```

Comment:
🤔 Cython should be getting installed, unless the version of `pip` is really old? (<18 I think)

Reply:
are we testing old versions of pip? we should install Cython to install dd-trace-py package, isn't it? 

the correct command would be:
```
install_command=python -m pip install {opts} {packages}
```

Is that your suggestion? :+1:

Reply:
yes, that _should_ be the correct command. I'd be curious if it didn't work.

Reply:
I think the problem is that we should pass "--no-binary" to each "deps" installation, instead to install gevent globally in "install_command". WIP